clang/include/clang/AST/DeclCXX.h

```diff
@@ -2878,7 +2878,7 @@ class CXXDestructorDecl : public CXXMethodDecl {
   static CXXDestructorDecl *CreateDeserialized(ASTContext &C, GlobalDeclID ID);
 
   void setOperatorDelete(FunctionDecl *OD, Expr *ThisArg);
-  void setOperatorArrayDelete(FunctionDecl *OD, Expr *ThisArg);
+  void setOperatorArrayDelete(FunctionDecl *OD);
```

Comment:
This looks like unrelated changes, but the changes themselves are correct.

Reply:
Yeah, this is a drive-by removal of unused argument added by a patch that caused the bug this patch is fixing.

Reply:
Should I do that in a separate commit?

Reply:
No, I think it's related enough as a drive-by it's fine.